science/pymol/Portfile

```diff
@@ -67,16 +68,17 @@ universal_variant   no
 patchfiles          pymol_shell.diff \
                     pmg_tk_platform.patch \
                     apbs-psize.patch  \
-                    pdb2pqr.patch
+                    pdb2pqr.patch \
+                    clang_opt_fix.diff
 
 require_active_variants tcl "" corefoundation
 require_active_variants tk "" quartz
 
 post-patch {
-    reinplace  "s|@PREFIX@|${prefix}|g" ${worksrcpath}/setup.py ${worksrcpath}/modules/pmg_tk/startup/apbs_tools.py
+    reinplace  "s|@PREFIX@|${prefix}|g" ${worksrcpath}/modules/pmg_tk/startup/apbs_tools.py
+    reinplace  "s|'/sw', '/opt/local'|'${prefix}'|g" ${worksrcpath}/setup.py
```

Comment:
Sorry, I really don't see the advantage of moving to a patch hunk with @PREFIX@ instead of just reinplacing on the complete text for "'/sw', '/opt/local'". The setup.py file tends to see the most churn of anything in pymol so the context constantly changes. it seems easier to deal with this with updates to the repinplace line than to be constantly regenerating a patch file.

Reply:
I asked the same question when I first came on the MacPorts scene a couple of years ago. The reinplace has the potential to do unpredictable things if the file you're reinplacing changes, and you might never know about it. If you patch to a placeholder and reinplace that, the chance of something unexpected happening becomes much smaller. 

It may not happen very often that that actually matters -- but some of these MacPorts guys have been doing this for 10 years +, and I guess they've seen issues with it often enough to have that as a strong preference / policy. It is an extra step, albeit an easy one. It's not a totally arbitrary nitpicky thing. I know homebrew doesn't bother with this, and does `inreplace` all over the place. 

If it's a significant hiccup for you, I can do it for you.